VideoLocker/src/main/java/org/edx/mobile/view/NavigationFragment.java

```diff
@@ -172,7 +173,7 @@ public void onClick(View v) {
                 Activity act = getActivity();
                 ((BaseFragmentActivity) act).closeDrawer();
 
-                if (!(act instanceof MyVideosTabActivity)) {
+                if (!(act instanceof MyVideosActivity)) {
```

Comment:
Sounds good

On Tue, Jun 7, 2016 at 4:37 PM, Christopher Lee <notifications@github.com>
wrote:

> In VideoLocker/src/main/java/org/edx/mobile/view/NavigationFragment.java
> <https://github.com/edx/edx-app-android/pull/733#discussion_r66147931>:
>
> > @@ -172,7 +173,7 @@ public void onClick(View v) {
> >                  Activity act = getActivity();
> >                  ((BaseFragmentActivity) act).closeDrawer();
> >
> > -                if (!(act instanceof MyVideosTabActivity)) {
> > +                if (!(act instanceof MyVideosActivity)) {
>
> Out of scope. Navigation drawer is another animal for another time.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/edx/edx-app-android/pull/733/files/b56764396a107f72839f6bab9e09873e45f5df61#r66147931>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AR10T4HQn-UScfxab_aR2sWomDzDw1jfks5qJdacgaJpZM4IvKlG>
> .
>



-- 
Regards,

Marc DiNino
Android Engineer
EDX